tests/test_unit/test_filtering.py

```diff
@@ -10,115 +10,235 @@
     savgol_filter,
 )
 
+# Dataset fixtures
+list_valid_datasets_without_nans = [
+    "valid_poses_dataset",
+    "valid_bboxes_dataset",
+]
+list_valid_datasets_with_nans = [
+    f"{dataset}_with_nan" for dataset in list_valid_datasets_without_nans
+]
+list_all_valid_datasets = (
+    list_valid_datasets_without_nans + list_valid_datasets_with_nans
+)
+
+
+# Expected number of nans in the position array per individual,
+# for each dataset
+expected_n_nans_in_position_per_indiv = {
```

Comment:
Very minor quibble, but since this is only used for the filtering test, should it be defined within that test, or maybe just above it?